docs/en/pact-functions.md

```diff
@@ -2005,6 +2005,17 @@ pact> (env-hash (hash "hello"))
 ```
 
 
+### env-in-repl {#env-in-repl}
+
+*in-repl*&nbsp;`bool` *&rarr;*&nbsp;`string`
+
+
+Set the in-repl flag, to potentially observe behaviors outside of the repl
```

Comment:
This description makes no sense to me. What does this do, and why would I set something called "in-repl" to true in order to observe behaviors "out of repl"?